certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+            },
+            "contact_customUserId": {
+              "type": "string",
+              "description": "contact_customUserId",
+              "title": "Contact Gratavid Custom User Id"
+            },
+            "contact_customAccountId": {
+              "type": "string",
+              "description": "contact_customAccountId",
+              "title": "Contact Gratavid Custom Account Id"
+            },
+            "contact_email": {
+              "type": "string",
+              "description": "contact_email",
+              "title": "Contact Email"
+            },
+            "contact_firstName": {
+              "type": "string",
+              "description": "contact_firstName",
+              "title": "Contact First Name"
+            },
+            "contact_lastName": {
+              "type": "string",
+              "description": "contact_lastName",
+              "title": "Contact Last Name"
+            },
+            "replyText": {
+              "type": "string",
+              "description": "replyText",
+              "title": "Reply Message"
+            },
+            "replyVideoSrc": {
+              "type": "string",
+              "description": "replyVideoSrc",
+              "title": "Video Reply Src"
+            }
+          }
+        }
+      },
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "label",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "sample": {
+                        "type": "string",
+                        "description": "sample"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Gratavid Data",
+        "description": "Get Gratavid notes and users",
+        "operationId": "getData",
+        "parameters": [
+          {
+            "name": "source",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "microsoftFlow",
```

Comment:
@Wtrapp Please note that the outdated product names can not be used anywhere in the connector artifacts.